cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonUtils.java

```diff
@@ -161,6 +162,16 @@ public static String getHumanReadable(long ts, boolean addUTC) {
         return humanRedable;
     } // getHumanRedable
     
+    public static long getMilliseconds(String timestamp) throws java.text.ParseException {
+        /*
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
Right! Fixed in PR https://github.com/telefonicaid/fiware-cygnus/pull/1374/commits/0743834aa54aa3fe114aad0ac3465e685f1f2145